### ISSUE TITLE
With AIL installed, Firefox extension KeePassXC-Browser cannot connect to KeePassXC if Firefox is launched AFTER KeePassXC

### ISSUE THREAD

**Issue by mitohund** [TheAssassin/AppImageLauncher#552]
### Pre-submit checks

- [X] I [checked for similar issues](https://github.com/TheAssassin/AppImageLauncher/issues?q=) beforehand, but could not find any, not even closed ones. I could not add my bug report to any existing issue.
- [X] I am going to take the time to to fill in all the required details. I know that the bug report will be dismissed otherwise.

### Describe the bug

I first filed this bug with the KeePassXC-Browser people ([here](https://github.com/keepassxreboot/keepassxc-browser/issues/1802)), and after a longish hunt for the cause I ended up with AIL seeming to be the culprit.

The issue: as long as AIL is installed, KeePassXC-Browser can only connect to KeePassXC if Firefox is launched AFTER KeePassXC. If Firefox is already running and I launch KeePassXC, no connection can be established.

This can be worked around by restarting Firefox with KeePaxxXC staying open. But that's quite inconvenient and sometimes not possible. The even more intrusive workaround is to uninstall AIL.

### Expected behavior

I would expect the KeePassXC to work no matter which application is launched first, Firefox or KeePassXC. AIL should not affect this.

### Steps to reproduce the issue

1. with AIL installed, launch Firefox
2. launch KeePassXC
3. click on KeePaxxXC-Browser extension -> does not work

### Distribution and desktop environment

KeePassXC - 2.7.4 AppImage
KeePassXC-Browser - 1.8.3.1
Operating system: Linux - Debian Sid
Browser: Firefox 108.0

### Installed AppImageLauncher version

appimagelauncher_2.2.0-gha111.d9d4c73+bionic_amd64

### List of AppImages you tried

KeePassXC-2.7.4-x86_64_2203fb14036f8806d01a903020aebd71.AppImage

**Comment by TheAssassin** [comment]
So you're saying this is working as expected with that same AppImage but with AppImageLauncher uninstalled?

**Comment by mitohund** [comment]
Yes, exactly. I have since switched to the flatpak of KeePassXC because I could  not get it to work.
